tests/test-opencode-smoke.sh

```
@@ -0,0 +1,183 @@
+  fi
+else
+  echo "  FAIL: init --opencode exited non-zero"
+  echo "$init_out"
+  exit 1
+fi
+echo ""
+
+# 12. session-start.sh hook with METASWARM_PLATFORM=opencode in a configured project
+echo "--- [12/12] Verify hooks/session-start.sh runs for opencode platform ---"
+HOOKTMP=$(mktemp -d /tmp/metaswarm-opencode-hook-XXXXXX)
+trap "rm -rf '$TMPDIR' '$SHTMP' '$HOOKTMP'" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)